apis/kms/v1alpha1/referencers.go

```diff
@@ -80,6 +80,9 @@ func ServiceAccountMemberName() reference.ExtractValueFn {
 		if !ok {
 			return ""
 		}
+		if n.Status.AtProvider.Email == "" {
```

Comment:
Future improvement: this functionality could be moved to live with ServiceAccount so that it doesn’t have to be repeated by each referencer and is at less risk of diverging from ServiceAccount API.

Reply:
Good point and it looks like it is not that hard, done :)